auditLogMover/package.json

```diff
@@ -5,23 +5,24 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.11.1",
+    "@typescript-eslint/parser": "^4.11.1",
     "aws-sdk-mock": "^5.1.0",
-    "jest": "^25.1.0",
-    "serverless": "^1.72.0",
-    "serverless-plugin-typescript": "^1.1.9",
-    "serverless-step-functions": "^2.19.0",
-    "sinon": "^9.0.2",
-    "ts-jest": "^25.1.0",
-    "typescript": "^3.7.5",
-    "@typescript-eslint/eslint-plugin": "^2.18.0",
-    "@typescript-eslint/parser": "^2.18.0",
     "eslint": "^6.8.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-import": "^2.20.0",
-    "eslint-plugin-prettier": "^3.1.2"
+    "eslint-plugin-prettier": "^3.1.2",
+    "jest": "^26.6.3",
+    "serverless": "^1.72.0",
+    "serverless-plugin-typescript": "^1.1.9",
+    "serverless-step-functions": "^2.19.0",
+    "sinon": "^9.0.2",
+    "ts-jest": "^26.4.4",
+    "typescript": "^4.1.3"
   },
   "dependencies": {
+    "@types/jest": "^26.0.19",
```

Comment:
Needed in dependencies?

Reply:
no. I just moved it to dev dependencies.

Turns out that `yarn upgrade` also installs packages if not already installed